src/webgl/p5.RendererGL.js

```diff
@@ -631,7 +635,22 @@ p5.RendererGL.prototype._getColorShader = function () {
   return this._defaultColorShader;
 };
 
+p5.RendererGL.prototype.getTexture = function (img) {
+  // efficiency?
+  var f = function(element) {
```

Comment:
Ohhh this is just hitting the limits of my experience with Javascript. I'd rather have it behave this way, but thought I couldn't use an object like p5.Image as the map key.